### DISCUSSION TITLE
Latest QMK features along with Miryoku after QMK user space

### DISCUSSION THREAD

**Discussion by WingHaa** [manna-harbour/miryoku#318] [Q&A] [answered]
I want to use handedness by prom feature which doesn't exists in QMK when users directory is still a thing. Is the only way prepare config.h and keymap.c myself from miryoku source for now?

**Comment by manna-harbour** [comment, marked as answer]
Handedness by EEPROM is supported by the version of QMK in Miryoku QMK: https://github.com/manna-harbour/miryoku_qmk/blob/miryoku/docs/feature_split_keyboard.md#handedness-by-eeprom.

To use Miryoku QMK with the current version of QMK, see https://github.com/manna-harbour/miryoku/discussions/287 (recently updated).
